### PR TITLE
LPS-167884 [Raylife AP] Add View of Underwriter Comments in Applications Details Page

### PR DESCRIPTION
**Pull request by joanagrego**
Related with --> [LPS-167884](https://issues.liferay.com/browse/LPS-167884)

### CODE REVIEW

**Comment by joanagrego** [comment]
ci:test:sf

**Comment by joanagrego** [comment]
ci:test:relevant

**Comment by joanagrego** [comment]
ci:reevaluate:1407401_8

**Comment by joanagrego** [comment]
ci:test:relevant

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/pages/ApplicationDetails/components/status-content/reviewed-rejected-content/index.tsx [hunk 1]
```diff
@@ -49,6 +60,32 @@ const UnderwritingContent = ({
...
+				}
+			);
+
+			const comment = message[0]?.message?.split(': ')[1];
+
+			setUnderwriterComment(comment);
+		});
+	}, [id]);
+
+	const handleShowComment = () => {
```
Comment (claricebrasil) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/pages/ApplicationDetails/components/status-content/reviewed-rejected-content/index.tsx:81]: I think you just need one validation here. 
```suggestion
	const handleShowComment = () => { underwriterComment ? underwriterComment : "This is uninsurable"}
```

**Review by claricebrasil — CHANGES_REQUESTED** [review]

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/pages/ApplicationDetails/components/status-content/reviewed-rejected-content/index.tsx (hunk 1 above)
Comment (joanagrego) [reply modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/pages/ApplicationDetails/components/status-content/reviewed-rejected-content/index.tsx:81]: In this case, this validation is necessary.

**Comment by joanagrego** [comment]
ci:forward

**Pull request closed without merging**